crates/resolver/src/caching_client.rs

```diff
@@ -481,6 +481,7 @@ where
     }
 }
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
We strive to keep each commit passing CI (even if we don't exactly always check this), so this commit should be squashed into the previous one. Given the size of the previous commit, it would make sense to make sure it can pass the `local-ci` stuff at least.